sqlglot/dialects/duckdb.py

```diff
@@ -146,6 +147,44 @@ def _build_greatest(args: t.List) -> exp.Greatest:
     return exp.Greatest(this=seq_get(args, 0), expressions=args[1:])
 
 
+def _approx_quantiles_sql(self: DuckDB.Generator, expression: exp.ApproxQuantiles) -> str:
```

Comment:
Nit: no need to put this in the global namespace. You can move it under `DuckDB.Generator` and rename into `approxquantiles_sql`.